go.mod

```diff
@@ -3,6 +3,7 @@ module ddns-updater
 go 1.12
 
 require (
+	github.com/BurntSushi/toml v0.3.1 // indirect
```

Comment:
Why do you need `github.com/BurntSushi/toml`? I don't see it used anywhere I think.

Reply:
I just run `go mod tidy` and I don't known where i used, before i push the pr it is already in the go.mod